### PR TITLE
Sg2 12 insurance policy

### PR DESCRIPTION
**Pull request by Jake-AS**
Pls reivew.

### CODE REVIEW

File: app/validators/insurance_policy.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import { TableName } from "app/enums";
+import { insurance_policy } from "../models";
+import Validator, { Type } from "./validator";
```
Comment (deanylev) [inline app/validators/insurance_policy.ts]: This should all be single quotes, not double

File: app/validators/insurance_policy.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+class InsurancePolicyValidator extends Validator<insurance_policy> {
```
Comment (deanylev) [inline app/validators/insurance_policy.ts]: The generic argument here should be `InsurancePolicy`

File: app/models.ts [hunk 3]
```diff
@@ -33,3 +33,14 @@ export interface User {
   password_hash: string;
   role: Role;
 }
+
+export interface insurance_policy {
```
Comment (deanylev) [inline app/models.ts]: `InsurancePolicy`

File: app/validators/insurance_policy.ts (hunk 1 above)
Comment (Jake-AS) [reply app/validators/insurance_policy.ts]: fixed.

File: app/models.ts (hunk 3 above)
Comment (Jake-AS) [reply app/models.ts]: fixed.

File: app/validators/insurance_policy.ts (hunk 2 above)
Comment (Jake-AS) [reply app/validators/insurance_policy.ts]: fixed.

**Pull request closed without merging**